juniper_official/Chassis/chassis-alarms.rule

```diff
@@ -89,6 +89,10 @@
             rule-properties {
                 version 2;
                 contributor juniper;
+                category basic;
+                is-scaling-rule {
```

Comment:
This is wrong. Scaling required only for suppose to be scaling rules. Revisit all rules and modify as shown below;

e.g.
Processes-"Fields:[ daemon pid ]; Directly impacted by number of processes running in each network device. Average 150+ processes in junos"
Interfaces
BGP
OSPF neighbors
ISIS adjacencies
L3VPNs
EVPNs
LSPs
LLDP
RIB Route-summary
FIB
SFLOW
RSVP sessions
Subscriber services - Check

Reply:
Made all the changes.